pkgs/development/misc/rappel/default.nix

```diff
@@ -0,0 +1,34 @@
+{ clang
+, fetchFromGitHub
+, libedit
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  name = "rappel";
```

Comment:
`name` should include a version, you can replace this with e.g.

```
  pname = "rappel";
  version = "unstable-2019-07-08";
```

which will set `name` to `${pname}-${version}`.